
package hospital.models;
import java.util.List;
import java.util.ArrayList;
import java.time.LocalDate;

public class Prescription {
    private int id;
    private Patient patient;
    private Doctor doctor;
    private LocalDate prescriptionDate;
    
    private List<PrescriptionItem> items = new ArrayList<>();
    
    public void addItem(PrescriptionItem item){
        items.add(item);
    }
    
    public void removeItem(PrescriptionItem item){
        items.remove(item);
    }
    
    public List<PrescriptionItem> getItems(){
        return items;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }
    
    

    public Patient getPatient() {
        return patient;
    }

    public void setPatient(Patient patient) {
        this.patient = patient;
    }

    public Doctor getDoctor() {
        return doctor;
    }

    public void setDoctor(Doctor doctor) {
        this.doctor = doctor;
    }

    public LocalDate getPrescriptionDate() {
        return prescriptionDate;
    }

    public void setPrescriptionDate(LocalDate prescriptionDate) {
        this.prescriptionDate = prescriptionDate;
    }
    
    
}
